server/libs/database/src/entities/album.entity.ts

```diff
@@ -0,0 +1,27 @@
+import { Column, CreateDateColumn, Entity, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { AssetAlbumEntity } from './asset-album.entity';
+import { UserAlbumEntity } from './user-album.entity';
+
+@Entity('shared_albums')
```

Comment:
We will need to create migration file for this as well

Reply:
Yep, I left that to the end, and then I forgot to do it 😅 . I have created a migration file including the rename of `shared_albums` to `albums` and `asset_shared_album` to `asset_album`